core/src/model/drawing.rs

```diff
@@ -88,45 +107,27 @@ pub fn export_drawing(
             .get(&stroke.color)
             .ok_or_else(|| CoreError::Unexpected(String::from("unable to get color from alias")))?;
 
-        if stroke.points_x.len() != stroke.points_y.len()
-            || stroke.points_y.len() != stroke.points_girth.len()
-        {
-            return Err(CoreError::Unexpected(String::from("unequal points and girth metrics")));
-        }
-
-        if stroke.alpha > 1.0 || stroke.alpha < 0.0 {
-            return Err(CoreError::Unexpected(String::from("invalid alpha value")));
-        }
-
         for point_index in 0..stroke.points_x.len() - 1 {
             let mut pb = PathBuilder::new();
             let x1 = stroke
                 .points_x
                 .get(point_index)
-                .ok_or_else(|| {
-                    CoreError::Unexpected(String::from("unable to get color from alias"))
-                })?
+                .ok_or_else(|| CoreError::Unexpected(String::from("unable to get stroke x1")))?
```

Comment:
@SmailBarkouch / @tvanderstad why not `CoreError::DrawingInvalid`?

Reply:
done!